### PR TITLE
VAGOV-1644 add news_story detail pages

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
- Add the news_story pages

## Testing done
- Locally with staging data

## Screenshots
screen shot 2019-02-25 at 12 01 50 pm

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/news_story.drupal.liquid [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                    </div>
+                    <div class="usa-grid usa-grid-full">
+                        <div class="va-introtext">
+                            <p>{{ fieldIntroText }}</p>
+                        </div>
+                        <div class="full-story">
+                            {{ fieldFullStory.processed }}
+                        </div>
+                    </div>
+                    <a href="#">See all community stories</a>
```
Comment (ahay-agile6) [inline src/site/layouts/news_story.drupal.liquid]: href: separate story to make news stories listing page

File: src/site/layouts/news_story.drupal.liquid [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+{% if header == empty %}
+    {% assign header = "h4" %}
+{% endif %}
+{% include "src/site/includes/header.html" %}
+{% include "src/site/includes/preview-edit.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+<div id="content" class="interior">
+    <main class="va-l-detail-page">
+        <div class="usa-grid usa-grid-full">
+            <nav class="usa-width-one-fourth va-sidebarnav" id="va-detailpage-sidebar"></nav>
```
Comment (jbalboni) [inline src/site/layouts/news_story.drupal.liquid]: Do these pages not have a sidebar? Should we remove this empty element?

**Review by ncksllvn — APPROVED** [review]
Agree with @jbalboni's comment about the sidebar but otherwise LGTM! Great work! 🎸💥

File: src/site/layouts/news_story.drupal.liquid (hunk 2 above)
Comment (ahay-agile6) [reply src/site/layouts/news_story.drupal.liquid]: they do have a sidebar... i just don't think it's made yet - part of another story. should i get rid of this for now?

Comment (ahay-agile6) [reply src/site/layouts/news_story.drupal.liquid]: i removed this

**Pull request merged**